lib/meadow/batch_driver.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Meadow.BatchDriver do
+  @moduledoc """
+  IntervalTask to kickoff queued batch update/delete jobs
+  """
+  import Ecto.Query, warn: false
+  import Meadow.Utils.Logging
+
+  alias Meadow.Batches
+  alias Meadow.Data.Schemas.Batch
+  alias Meadow.IntervalTask
+  alias Meadow.Repo
+
+  use IntervalTask, default_interval: 30_000, function: :drive_batch
+
+  require Logger
+
+  @timeout 60
+
+  @doc """
+  If no batches are currently running, find the next one and start it
+  """
+  def drive_batch(state) do
+    # todo - check for stale batches exceeding some extreme upper bounds timeout
+
+    case Batches.next_batch() do
+      nil ->
+        Logger.info("No batches queued")
```

Comment:
It's a fine line with logging always, but my take here is this adds extra noise to the log. Would maybe just do `nil -> :noop` instead